instrumentation/kamon-redis/src/main/scala/kamon/instrumentation/jedis/JedisInstrumentation.scala

```diff
@@ -8,27 +8,29 @@ import redis.clients.jedis.commands.ProtocolCommand
 
 class JedisInstrumentation extends InstrumentationBuilder {
   onType("redis.clients.jedis.Protocol")
-    .advise(method("sendCommand").and(withArgument(1, classOf[ProtocolCommand])), classOf[SendCommandAdvice])
+    .advise(method("sendCommand"), classOf[SendCommandAdvice])
 }
 
 class SendCommandAdvice
-
 object SendCommandAdvice {
   @Advice.OnMethodEnter()
-  def enter(@Advice.Argument(1) command: ProtocolCommand) = {
-    val spanName = s"redis.command.${command}"
-    val span = Kamon.clientSpanBuilder(spanName, "redis.client.jedis")
-      .start()
-
-    span
+  def enter(@Advice.Argument(1) command: Any) = {
```

Comment:
I'm not 100% sure that mentions of the `ProtocolCommand` type must be removed from here. Have you tried keepting this method the same as it was before?

Reply:
It's there cause there's two methods called sendCommand with 3 arguments. If I leave it as ProtocolCommand, this advice will try to instrument both methods, and fail when the second argument isn't a protocol command
![image](https://user-images.githubusercontent.com/13893057/119961001-db055180-bfa5-11eb-8243-a5a3ad6b781b.png)

Reply:
I'll revert the advice to test whether it fails

Reply:
It fails if I leave ProtocolCommand in the signature: 
```
ERROR 2021-05-28 11:20:41  Logger : Error => redis.clients.jedis.Protocol with message 
Cannot assign [B arg1 to interface redis.clients.jedis.commands.ProtocolCommand. 
Class loader: sun.misc.Launcher$AppClassLoader@18b4aac2: 
java.lang.IllegalStateException: Cannot assign [B arg1 to interface redis.clients.jedis.commands.ProtocolCommand
```

Reply:
Got it, then it goes in as `Any` :muscle: